### PR TITLE
fix/euler_radians_to_degrees

### PR DESCRIPTION
**Pull request by marija-p**
Fix node to convert from radians to degrees. Tested with colibri_gps.launch.

### CODE REVIEW

**Comment by marija-p** [comment]
@enricgalceran

File: geodetic_utils/src/euler_radians_to_degrees.py [hunk 1]
```diff
@@ -8,20 +8,20 @@
 from math import *
 import sys
 
-class EulerDegreestoRadians:
+class EulerRadianstoDegrees:
```
Comment (enricgalceran) [inline geodetic_utils/src/euler_radians_to_degrees.py]: Casing.

**Comment by ffurrer** [comment]
@marija-p why are you committing as @fmina ?

**Comment by fmina** [comment]
is it my account on flybook?

**Comment by marija-p** [comment]
That's because Mina's account is logged into Colibri. We've discussed this just now and will clear the e-mails on the MAVs and laptops. :)

**Comment by enricgalceran** [comment]
LGTM, already addressed identity theft apart.

**Pull request merged**